Packs/Vectra_AI/Integrations/VectraDetect/VectraDetect.py

```diff
@@ -0,0 +1,1559 @@
+                                     f"Skipping other objects.")
+                        break
+
+                    if entity_type == 'Accounts':
+                        incident, incident_last_run = account_to_incident(event)
+                    elif entity_type == 'Hosts':
+                        incident, incident_last_run = host_to_incident(event)
+                    elif entity_type == 'Detections':
+                        incident, incident_last_run = detection_to_incident(event)
+
+                    if (incident_last_run.get('last_timestamp') == last_fetched_timestamp) \
+                            and (incident_last_run.get('id') == last_fetched_id):
+                        # Start creating incidents after this one as already fetched during last run
+                        start_ingesting_incident = True
+                        continue
+
+                    if last_fetched_incident_found and not start_ingesting_incident:
+                        demisto.debug(f"{entity_type} - Skipping object "
+                                      f"last_timestamp : {incident_last_run.get('last_timestamp')} "
+                                      f"/ ID : {incident_last_run.get('id')}")
+                    else:
+                        demisto.debug(f"{entity_type} - New incident from object "
+                                      f"last_timestamp : {incident_last_run.get('last_timestamp')} "
+                                      f"/ ID : {incident_last_run.get('id')}")
+                        entity_incidents.append(incident)
+                        new_last_run[entity_type] = incident_last_run
+
+            if len(entity_incidents) > 0:
+                demisto.info(f"{entity_type} - {len(entity_incidents)} incident(s) to create")
+                incidents += entity_incidents
+            else:
+                demisto.info(f"{entity_type} - No new incidents to create, keeping previous last_run data")
+                new_last_run[entity_type] = previous_last_run[entity_type]
+
+            # Update remaining list
+            remaining_fetch_types.remove(entity_type)
+            if len(remaining_fetch_types) > 0:
+                max_created_incidents = (sanitize_max_results(integration_params.get('max_fetch')) - len(incidents)) \
+                    // len(remaining_fetch_types)
+
+    demisto.info(f"{len(incidents)} total incident(s) to create")
+
+    return new_last_run, incidents
+
+
+def vectra_search_accounts_command(client: Client, **kwargs) -> CommandResults:
+    """
+    Get several accounts objects maching the search criterias passed as arguments
+
+    :return:
+        A ``CommandResults`` object that is then passed to ``return_results``,
+        that contains Accounts
+    :rtype: ``CommandResults``
+    """
+
+    api_response = client.search_accounts(**kwargs)
+
+    count = api_response.get('count')
+    if not count:
+        VectraException('Cannot find any account')
```

Comment:
I've reworked this part.
It will raise a VectraException if the 'count' attribute isn't present in the API JSON response
However if count = 0, it will return a human readable message as suggested